packages/wp-checkout/src/components/basics.js

```diff
@@ -84,8 +78,10 @@ export const StepHeader = styled.h2`
 export const StepTitle = styled.span`
 	font-weight: ${props =>
 		props.isActive ? props.theme.weights.normal : props.theme.weights.bold};
-	color: ${props => ( props.isActive ? props.theme.colors.gray80 : props.theme.colors.black )};
+	color: ${props => ( props.isActive ? props.theme.colors.black : props.theme.colors.gray80 )};
 	margin-right: 5px;
+	font-weight: ${props =>
```

Comment:
There's already a `font-weight` property just a few lines up. I think we can just remove that one.